src/NBitcoin/BitcoinCore/Coins.cs

```diff
@@ -5,132 +5,138 @@ namespace NBitcoin.BitcoinCore
 {
     public class Coins : IBitcoinSerializable
     {
-        bool fCoinStake;
+        private bool fCoinStake;
+        private uint nTime;
+        private uint nHeight;
+        private uint nVersion;
+
+        // Determines whether transaction is a coinbase.
+        public bool CoinBase { get; set; }
+
         public bool CoinStake
```

Comment:
Can you maybe collapse some of the extra lines here?

Reply:
Not sure what you mean. Only one blank line max to improve readability.